README.md

```diff
@@ -1,56 +1,91 @@
+Our inspiration for this is [Stow](https://www.gnu.org/software/stow/) a symlink farm manager. You can use Depot in combination with Stow to store your
+package list in the same place as your dotfiles.
 
 ## Installation
+
+### Pre-built binaries
+You can download the latest release from the [releases page](https://github.com/Rignchen/Depot/releases/latest).\
+After downloading the binary, you can move it to a directory in your PATH.
+
+### Package manager
+Package manager support is coming soon.\
+Consider building from source or using the pre-built binaries for now.
+
+### Building from source
+
 ```bash
 git clone git@github.com:Rignchen/Depot.git
 cd Depot
 cargo build --release
 mv target/release/depot /usr/local/bin
 ```
+Please note that you need to have [Rust](https://www.rust-lang.org/tools/install) installed to build the project.
 
 ## Configuration
-Depot will try to guess your package manager depending on your operating system, however sometimes you may want to specify it yourself, for example on Arch Linux you may want to use `yay` instead of `pacman`.\
+Depot will try to guess your package manager based on your operating system. However, sometimes you may want to specify it yourself. For example on Arch (btw) you may want to use `yay` instead of `pacman`.\
 To do so you have 2 options:
 1. Set the environment variable `DEPOT_PM` to the name of your package manager
 2. Add the flag `--pm <package_manager>` when running a command
 
 ## Usage
 Depot has 5 commands:
-- `install <package>`: Install a package
-- `uninstall <package>`: Uninstall a package
-- `update` <package>: Update a package, if no package is specified, update all the packages
-- `list`: List all the packages you have installed
-- `sync`: Compare the packages installed on the computer with the ones stored in the file and install the missing ones
+- `install <package>`: Install a package.
+- `uninstall <package>`: Uninstall a package.
+- `update [package]`: Update a package, if no package is specified, update all the packages.
+- `list`: List all the packages you have installed.
+- `sync`: Compare the packages installed on the computer with the ones stored in the file and install the missing ones.
+
 
 ## Example
-In this example I will install `neovim` and `zsh` on both my computers
+In this example I will install `vim` (btw) and `zsh` on both of my computers.\
+I use `scp` to copy the package list file from one computer to the other.\
+Then I use `ssh` to connect to the other computer and run `depot sync` to install the missing packages on the other computer.
 ```bash
-Depot install neovim
-Depot install zsh
-scp ~/.depot/packages user@computer:~/.depot/packages
-ssh user@computer
-Depot sync
+$ depot install vim
+vim is now installed.
+
+$ depot install zsh
+zsh is now installed.
+
+$ scp ~/.depot/packages user@computer:~/.depot/packages
+package list file copied to computer
+
+$ ssh user@computer 
+Welcome to computer.
```

Comment:
doesn't match the answer of ``ssh``
what you could do instead is
```bash
$ ssh user@computer
$ whoami
user
```
that way if someone wants to reproduce the example, the answers on his computer would be more similar to this readme